Doc/add examples

shtrom opened the pull request:
Some examples to match the PHP and Ruby SDKs.

Not sure whether we actually want examples in the SDK source, though.

amlearn commented:
we got rid of the makefile?

amlearn commented:
#derp... actually yeah, it's just the single commit

alangarf commented on examples/simple/init_data.py:
```diff
@@ -0,0 +1,45 @@
+        i,
+        json.dumps(item)
+        ))
+    if i > 5:
```
What is that `i` even doing? And same above, it's iterating on a iterable thing, why do we care about `i`?

alangarf replied:
Oh, it's in the print output.

```
for i, result in enumerate(client.request_iter(itembankUri, security, consumer_secret, data_request, action)):
```

shtrom replied:
🤦‍♂️

shtrom replied:
I is just an arbitrary limit so we don't get output forever.

shtrom commented:
Fixed, rebased, and bundled a couple of additional cosmetic fixups. You can test the examples with

```make
. .venv/Darwin-x86_64-sdk-python/bin/activate
./examples/simple/init_items.py
[...]
./examples/simple/init_data.py
[...]
```

stellalie commented:
@shtrom Thanks for that, we are bit swamped at the moment, and will get someone from Turtle take a look at this later at the end of release. Cheers!
